handbook/README.md

```diff
@@ -7,47 +6,49 @@ This policy does not form part of any employee&#39;s contract of employment and
 we may amend it at any time.
 
 ## Contents
-* Introduction
-  * [Introduction](#introduction)
-  * [Using the Staff Handbook](#using-the-staff-handbook)
-  * [Responsibility for the Staff Handbook](#responsibility-for-the-staff-handbook)
-* Best Practice
-  * [Anti-corruption and Bribery Policy](best-practice/anti-corruption.md)
-  * [Anti-harassment and Bullying Policy](best-practice/anti-harassment.md)
-  * [Health and Safety Policy](best-practice/health-and-safety-policy.md)
-  * [Whistle-blowing Policy](best-practice/whistle-blowing-policy.md)
-* HR
-  * [Compassionate Leave](hr/compassionate-leave.md)
-  * [Disciplinary and Capability Procedure](hr/disciplinary-procedure.md)
-  * [Dress Code](hr/dress-code.md)
-  * [Equal Opportunities Policy](hr/equal-opportunities.md)
-  * [Expenses Policy](hr/expenses-policy.md)
-  * [Grievance Procedure](hr/grievance-procedure.md)
-  * [Maternity Policy](hr/maternity-policy.md)
-  * [Off-boarding](hr/off-boarding.md)
-  * [On-boarding](hr/on-boarding.md)
-  * [Organisation Chart](hr/org-chart.md)
-  * [Paternity Policy](hr/paternity-policy.md)
-  * [Personal details, home address and next of kin](hr/personal-details.md)
-  * [Redundancy Policy](hr/redundancy-policy.md)
-  * [Referral Programme](hr/referral-programme.md)
-  * [Retirement Policy (no fixed retirement age)](hr/retirement-policy.md)
-  * [Risk Management Policy](information-security/risk-management.md)
-  * [Sickness Absence Policy](hr/absence.md)
-  * [Time Management Policy](quick-reference.md#time-management)
-* [Information Security](information-security/README.md)
-* Operations
-  * [Hiring](operations/greenhouse.md)
-  * [Interviewing](operations/interviewing.md)
-  * [Invoicing](operations/invoicing.md)
-  * [Sales](operations/hubspot.md)
-* Professional Development
-  * [Developer Proficiency](professional-development/developer-proficiency.md)
-  * [External Training Courses](professional-development/external-training-courses.md)
-  * [Mentoring](professional-development/mentoring.md)
-  * [Goal Reviews](professional-development/goal-reviews.md)
-* [Quick-Reference Guide](quick-reference.md)
-* [Risk Management](information-security/risk-management.md)
+
+- Introduction
```

Comment:
I don't know why the original had `- *` before all the items instead of just `-`... VSCode has automatically fixed that 🤷‍♂